ts/Stock/StockToolsBindings.ts

```diff
@@ -533,18 +533,17 @@ bindingsUtils.updateNthPoint = function (
         annotation: Annotation
     ): void {
         const options = annotation.options.typeOptions,
-            coords = this.chart.pointer.getCoordinates(e),
-            coordsX = this.utils.getAssignedAxis(coords.xAxis),
-            coordsY = this.utils.getAssignedAxis(coords.yAxis);
+            xAxis = isNumber(options.xAxis) && this.chart.xAxis[options.xAxis],
+            yAxis = isNumber(options.yAxis) && this.chart.yAxis[options.yAxis];
```

Comment:
Are these options always `number|undefined` @pawelfus ? In other, related options, we can also apply the id of the associated axis as a string.

Reply:
`updateNthPoint()` is an internal method and what's defined in `options.xAxis` is also set by us. And this is always a `number` (index). In fact, we don't need `isNumber(options.xAxis)` check, but probably TS requires this